helix-core/src/main/java/org/apache/helix/monitoring/mbeans/ClusterStatusMonitor.java

```diff
@@ -1121,4 +1120,13 @@ public long getPendingStateTransitionGuage() {
     }
     return total;
   }
+
+  @Override
+  public long getRebalanceThrottledByErrorPartitionGauge() {
```

Comment:
The naming is confusing. It is called getRebalanceThrottledByErrorPartitionGauge both in ClusterStatusMonitor and in ResourceMonitor. You may consider something like numOfResourcesRebalanceThrottledGauge, or some other name you think is more clear.